spec/services/order_filler_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe PurchasesHandler::OrderFiller, type: :service do
+
+  describe "#call" do
+
+    context "when it is successful" do
```

Comment:
Faltou testar o caso de falha. O que aconteceria se os dados passados fossem inválidos?

Reply:
isso ta sendo tratado no controller. se a order não for válida ele retorna erro. devo incluir isso no teste do service tb?

Reply:
@jesrodri na verdade, realmente não precisa, quando eu vi esse context pensei que havia alguma condicional dentro do service.